src/transformation/Transformation.ts

```diff
@@ -362,11 +352,11 @@ class Transformation {
    * @param {IVideoEditAction} videoEditAction
    * @return {this}
    */
-  /*
-  videoEdit(videoEditAction: VideoEditAction): this {
-    return this.addAction(videoEditAction);
+
+  videoEdit(action: IAction): this {
```

Comment:
Can this accept a videoEditAction instead?
can all those actions have a common base type?